flink-runtime/src/test/java/org/apache/flink/runtime/resourcemanager/slotmanager/SlotManagerFailUnfulfillableTest.java

```diff
@@ -52,6 +53,9 @@
  */
 public class SlotManagerFailUnfulfillableTest extends TestLogger {
 
+	private static final WorkerResourceSpec WORKER_RESOURCE_SPEC = new WorkerResourceSpec(
+		100.0, 10000, 10000, 10000, 10000);
```

Comment:
Looking at this line, I would recommend introducing a builder for the `WorkerResourceSpec` and to make the default constructor private.